pages/index.js

```diff
@@ -1,7 +1,28 @@
+// lib imports
```

Comment:
```suggestion
```
I think this is clear, so you don't need that. You could reread the code review guid in the capstone docs, for which comments are good and which are more disturbing than helpful for the readability.